src/scenario-registry.ts

```diff
@@ -233,6 +238,31 @@ export const scenarios = [
     displayOrder: 8,
     rootComponent: ListUserPaymentHistory,
   },
+
+  /**
+   * Personalization related scenarios
+   */
+  {
+    groupName: 'Personalization',
+    shortId: 'array-get-favorites',
+    displayName: 'Array: Get Favorites',
```

Comment:
Let's stick with one standard here for all 3 scenarios - Favorites or Favorite(s)

Reply:
As discussed will skip this change